### DISCUSSION TITLE
Could not find the TLS certificate file

### DISCUSSION THREAD

**Discussion by rajatanand86** [aws/aws-iot-device-sdk-python#322] [Q&A] [answered]
I am trying to run this code https://docs.aws.amazon.com/iot/latest/developerguide/http.html in ret.py with arguments as given below.

python ret.py --endpoint a1ifr8g4k1agsb-ats.iot.us-east-1.amazonaws.com --cert "C://Users//DELL//Desktop//sdfsdf-certificate.pem" --key "C://Users//DELL//Desktop//sdfsdf-private.pem" --topic mytopic --message "hello"

and getting below error.
My file is in the folder as given.
OSError: Could not find the TLS certificate file, invalid path: 

Can someone help?

**Comment by jmklix** [comment, marked as answer]
I think you need to check your file paths and names of the files that you are using. I was able to run the sample in powershell and git bash. Also make sure to check the file extensions; I see you are using `.pem` but they might look like this: `.pem.crt` and `.pem.key`

**Comment by bretambrose** [comment]
I see double forward slashes in Windows paths which seems suspect.

**Reply by jmklix** [reply]
I had tried a few different variations(`/`, `\`, `//`, relative paths) in powershell and they all seemed to work fine, but this is another variable that you should change when testing this @rajatanand86.

**Comment by rajatanand86** [comment]
Hi. It is working fine now. Thanks.
